10.9-libcxx/stable/main/finkinfo/base/libncurses6-shlibs.info

```diff
@@ -17,6 +17,9 @@ PatchScript: <<
 
 	#Need to remove these entries to avoid failure on case sensitive file systems
 	perl -pi -e "s/hp70092a\|//; s/hp2621a\|//; s/2621a\|//" misc/terminfo.src
+
+	# Turn this flag on to catch any issues
+	perl -pi -e 's|-Wno-error=implicit-function-declaration|-Werror=implicit-function-declaration|g' configure
```

Comment:
I think I'd just remove it entirely instead of replacing it? I mean, since it's the default now, anyways... idk, I wonder what @akhansen thinks?